test/testdata/lsp/code_actions/move_method/no_sig.A.rbedited

```diff
@@ -1,12 +1,11 @@
-# typed: strict
+# typed: true
 # selective-apply-code-action: refactor.extract
 #
 # The test is asserting we can move a method without a sig
 
 module BarModule
   extend T::Sig
   def self.bar
-# ^^^^^^^^^^^^ error: The method `bar` does not have a `sig`
```

Comment:
The way our test framework is that if there's a `apply-code-action` after a `error`, it assumes the code action is a quick fix for the error, and checks if the code action is return for the range the error corresponds to. In this case, the error and code action are unrelated.